crypto/md5.c

```diff
@@ -158,15 +158,22 @@ EXP_FUNC void STDCALL MD5_Final(uint8_t *digest, MD5_CTX *ctx)
 {
     uint8_t bits[8];
     uint32_t x, padLen;
-
+#ifdef FORCEL32_NOT_SUPPORTED
```

Comment:
Suggest that we come up with a better name for this macro... I bet most people reading this code won't know what `FORCEL32` is, so i suggest something like `CONST_ARRAYS_IN_PROGMEM`...